src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 40px;
+}
+
+.header_img {
+  align-items: center;
+  padding-top: 10px;
+}
+
+/* .nav {
+  align-items: center;
+} */
+
+.ul {
+  margin: 0;
+  display: flex;
+  padding: 22px;
+
+}
+
+.list {
+  list-style: none;
+  height: 38px;
+}
+
+.list:hover {
+  border-bottom: solid 4px;
+  color: #00acdc;
+}
+
+.is_active {
+  font-family: inherit;
```

Comment:
font-family is redudant here as you set it before

Reply:
Ok